docs/dbmodel.rst

```diff
@@ -49,3 +49,32 @@ In addition, a column can be set to auto-increment with the ``counters`` key wor
 This is used, for example, in the ``MotionCorrection`` table which contains an ``image_number`` column which 
 needs to be a unique integer for each micrograph. An appendable column may be specified with the ``append`` 
 keyword. For these columns values are appended to a list rather than updated.
+
+------
+DBNode
+------
+
+A ``DBNode`` is a ``Node`` implemented with a specific call. Calling a ``DBNode`` will perform an insert into 
+a ``Table`` associated with the node after doing a few checks. If ``"check_for`` is present in the node's 
```

Comment:
```suggestion
a ``Table`` associated with the node after doing a few checks. If ``check_for`` is present in the node's 
```
would like to note that the entire section is very dense :slightly_smiling_face: 

With the further description in the other file I believe this is sufficiently complex to benefit from docstrings and auto-generated documentation.

Reply:
I agree the documentation needs improving. I'll try and make some improvements in the next PR and then do some more once we have a better idea of how things will change once the Relion python interface is available